src/github.rs

```diff
@@ -451,42 +472,54 @@ pub async fn command_upload_release_distributions(args: &ArgMatches) -> Result<(
 
     let mut digests = BTreeMap::new();
 
-    let mut fs = vec![];
-
-    for (source, dest) in wanted_filenames {
-        if !filenames.contains(&source) {
-            continue;
-        }
-
-        let file_data = Bytes::copy_from_slice(&std::fs::read(dist_dir.join(&source))?);
+    let retry_policy = ExponentialBackoff::builder().build_with_max_retries(5);
+    let raw_client = ClientBuilder::new(reqwest_middleware::reqwest::Client::new())
+        .with(RetryTransientMiddleware::new_with_policy_and_strategy(
+            retry_policy,
+            GitHubUploadRetryStrategy,
+        ))
+        .build();
 
-        let mut digest = Sha256::new();
-        digest.update(&file_data);
+    {
+        let mut fs = vec![];
```

Comment:
~unchanged, needed to put this in a block to satisfy the borrow checker with borrow of `raw_client` in `fs`.